image/spec/v1.1.md

```diff
@@ -370,10 +370,7 @@ whitespace. It has been added to this example for clarity.
 
         <ul>
           <li>
-            <code>type</code> is usually set to <code>layers</code>. There is
-            also a Windows-specific value <code>layers+base</code> that allows
-            a base layer to be specified in a field of <code>rootfs</code>
-            called <code>base_layer</code>.
+            <code>type</code> is usually set to <code>layers</code>.
```

Comment:
We should probably leave this in or [omit from OCI](https://github.com/opencontainers/image-spec/blob/master/serialization.md#image-json-field-descriptions) if we don't intend to support it going forward.